content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -37,8 +37,10 @@ The upgrade workflow at high level is the following:
 
 ### Additional information
 
-- [Draining nodes](/docs/tasks/administer-cluster/safely-drain-node/) before kubelet MINOR version
-  upgrades is required. In the case of control plane nodes, they could be running CoreDNS Pods or other critical workloads.
+- The instructions below outline when to drain each node during the upgrade process.
+It is a required step before kubelet MINOR version upgrades. In the case of control plane nodes,
```

Comment:
optional further improvement:
```suggestion
If you are performing a **major** or **minor** version upgrade for any kubelet, you **must**
first drain the node (or nodes) that you are upgrading. In the case of control plane nodes,
```

Reply:
updated to include the text as you recommended but excluded `major`.
there is no such thing as a major version upgrade of the kubelet yet, and who knows what requirements it will have when a new major lands.